src/components/Contact/SocialMediaIcons.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import Facebook_logo from "../../img/img-icon-Facebook.svg";
+import Instagram_logo from "../../img/img-icon-Instagram.svg";
+import Twitter_logo from "../../img/img-icon-Twitter.svg";
```

Comment:
SocialMediaIcons.js: Code is clean, clearly presented, and well-organized for future additions/amendments. Any particular reason for the change from `import React, { Component } from 'react';` to `import React, { Component } from "react";`? Any particular reason for the change to `import Facebook_logo from "../../img/img-icon-Facebook.svg";`, `import Instagram_logo from "../../img/img-icon-Instagram.svg";`, and `import Twitter_logo from "../../img/img-icon-Twitter.svg";`?